### DISCUSSION TITLE
[@types/minio] The param type of setRequestOptions is inaccurate

### DISCUSSION THREAD

**Discussion by yucao2521** [DefinitelyTyped/DefinitelyTyped#63966] [Issues with a @types package]
Hi!

Here is the [function with issue](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fc41bdd10649879190f2997632d01f338e11c761/types/minio/index.d.ts#L441).

According to the [code](https://github.com/minio/minio-js/blob/4fce54bd81188aaa86fe6927786877cd71c2b79b/src/main/minio.js#L474) of `minio.js`, `options`, the param of `setRequestOptions`, will be finally passed to [`http.request`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fc41bdd10649879190f2997632d01f338e11c761/types/node/http.d.ts#L1548) or [`https.request`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fc41bdd10649879190f2997632d01f338e11c761/types/node/https.d.ts#L496). 
So it is most likely a `RequestOptions` rather than an `AgentOptions`.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
I just opened #64060 😄
